src/test/java/com/assignmentTest/Assignment4Test.java

```diff
@@ -0,0 +1,17 @@
+package com.assignmentTest;
+import org.junit.jupiter.api.Assertions;
+import com.assignment.Assignment4;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Assignment4Test {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void substring(){
+        Assignment4 Test = new Assignment4();
```

Comment:
It is a good practice to have the object name = class name but in lower case.
Eg : Assignment4 assignment4=new Assignment4()